README.md

```diff
@@ -1,113 +1,103 @@
 
-## Key features
+Genkit is built for developers adding generative AI to their apps with Node.js or Go, and can run anywhere these runtimes are supported. It works with any generative model API or vector database through [plugins](#plugin-ecosystem).
 
-### GenAI models
+Developed by the [Firebase](https://firebase.google.com) team, Genkit can be used independently of Firebase or Google Cloud services.
 
-**Unified API for generation** across AI models built by Google (Gemini, Gemma) and third party providers. Supports multimodal input, multimedia content generation, and custom options.
+## Get started
 
-**Generate structured output** as strongly-typed objects with custom schemas for easy integration into your app.
+- [Node.js quickstart](https://firebase.google.com/docs/genkit/get-started)
+- [Next.js quickstart](https://firebase.google.com/docs/genkit/nextjs)
+- [Go quickstart](https://firebase.google.com/docs/genkit-go/get-started-go)
 
-**Define custom tools for your AI models** to fetch data, display UI, write to a database, and more.
+> [!NOTE]
+> Genkit for Go is in alpha, so we only recommend it for prototyping.
 
-**Write well structured prompts** with all relevant metadata encapsulated in a single [dotprompt](https://firebase.google.com/docs/genkit/dotprompt) file. Supports handlebars templating, history, multimedia, and more.
+## Library key features
 
-![Screenshot of IDE showing Firebase Genkit RAG sample code](docs/resources/readme-rag-screenshot.png)
+- **Unified generation API:** Generate text, media, structured objects, and tool calls from any generative model using a single, adaptable API.
 
-### Retrieval
+- **Vector database support:** Add retrieval-augmented generation (RAG) to your apps with simple indexing and retrieval APIs that work across vector database providers.
 
-**Build context-aware AI features** by indexing your data and dynamically retrieving relevant information from your database. Genkit provides flexible, light-weight abstractions for indexers and retrievers that work with any database provider.
+- **Enhanced prompt engineering:** Define rich prompt templates, model configurations, input/output schemas, and tools all within a single, runnable [.prompt](https://firebase.google.com/docs/genkit/dotprompt) file.
 
-### Evaluation
+- **AI [work]flows:** Organize your AI app logic into [Flows](https://firebase.google.com/docs/genkit/flows) - functions designed for observability, streaming, integration with Genkit devtools, and easy deployment as API endpoints.
 
-**Evaluate your end-to-end AI workflow** using a variety of pre-built and custom evaluators. As easy as:
+- **Built-in streaming:** Stream content from your Genkit API endpoints to your client app to create snappy user experiences.
 
-```bash
-genkit eval:flow myAiWorkflow --input testQuestions.json
-```
+## Development tools
 
-### Extensibility with plugins
+Genkit provides a CLI and a web-based Studio to streamline your AI development workflow.
```

Comment:
👎 👎 👎 to Studio

Reply:
I suggest leaving as "local developer UI" or similar for now so as to add more to this change, and address the tool naming in a second PR